src/main/java/net/runelite/rsb/methods/Inventory.java

```diff
@@ -670,15 +671,15 @@ public int getSelected() {
 		Consumer<Image> imageCallback = (img) ->
 		{ // This callback is on the game thread, move to executor thread
 			{
-				methods.runeLite.getInjector().getInstance(ScheduledExecutorService.class).submit(()-> {
+				methods.runeLite.getInjector().getInstance(ScheduledExecutorService.class).submit(() -> {
 					selector.setSelected(getSelected(img, comps));
 					synchronized (selector) {
 						selector.notify();
 					}
 				});
 			}
 		};
-		methods.runeLite.getInjector().getInstance(DrawManager.class).requestNextFrameListener(imageCallback);
+		RuneLite.getInjector().getInstance(DrawManager.class).requestNextFrameListener(imageCallback);
```

Comment:
is there a reason these are different?

Reply:
I overlooked. Will be changed in next commit